src/e2e/java/teammates/e2e/pageobjects/CourseJoinConfirmationPage.java

```diff
@@ -0,0 +1,37 @@
+package teammates.e2e.pageobjects;
+
+import static org.junit.Assert.assertEquals;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Page Object Model for the course join confirmation page.
+ */
+public class CourseJoinConfirmationPage extends AppPage {
+    @FindBy(id = "btn-confirm")
+    private WebElement confirmButton;
+
+    public CourseJoinConfirmationPage(Browser browser) {
+        super(browser);
+    }
+
+    @Override
+    public boolean containsExpectedPageContents() {
+        if (isElementPresent(By.className("modal-content"))) {
+            return waitForElementPresence(By.tagName("h2")).getText().contains("Uh oh! Something went wrong.");
```

Comment:
This is not something specific to the course join page, though

Reply:
Removed.